toolbox/fdc3-compliance/src/test/fdc3.addContextListener.ts

```diff
@@ -0,0 +1,7 @@
+describe("fdc3.addContextListener", async () => {
```

Comment:
Do we need the top level async (this comment applies to all test files).

Reply:
It turns out that we don't need the top-level async, so will remove it.